src/Php54/Php54.php

```diff
@@ -39,4 +39,63 @@ public static function hex2bin($data)
 
         return $data;
     }
+
+    public static function http_response_code($responseCode)
+    {
+        static $currentCode = 200;
+
+        $messages = array(
+          100 => 'Continue',
+          101 => 'Switching Protocols',
+          200 => 'OK',
+          201 => 'Created',
+          202 => 'Accepted',
+          203 => 'Non-Authoritative Information',
+          204 => 'No Content',
+          205 => 'Reset Content',
+          206 => 'Partial Content',
+          300 => 'Multiple Choices',
+          301 => 'Moved Permanently',
+          302 => 'Moved Temporarily',
+          303 => 'See Other',
+          304 => 'Not Modified',
+          305 => 'Use Proxy',
```

Comment:
@keradus - 307/308 added.  FWIW, most servers ignore this value and use their own.  I've only been able to see these values using `php -S`